manifests/charts/base/values.yaml

```diff
@@ -25,3 +25,6 @@ base:
 
   # For istioctl usage to disable istio config crds in base
   enableIstioConfigCRDs: true
+
```

Comment:
See above - the goal was to keep base separated from the revisions.

I also suspect helm may be unhappy if different installs operate on the same file. Can you run some tests using helm install of the base ? 

Note that another approach is to just use 'validationURL' - and specify "istiod-rev.istio-system.svc/validate.

Reply:
Doing it this way does solve the "requires cluster admin" problem which I like, but doesn't solve the problem that the ValidatingWebHook is both cluster scoped and versioned.

Reply:
How does it solve  "requires cluster admin" ? To change make your revision the "default" you would need to update the VWH to the new url, right?

Reply:
Sorry, solves the "requires cluster admin for _every_ install instead of just installs where you update the default revision" problem.

Reply:
In the end - if we don't have floating revisions ( i.e. 'prod' pointing to various specific installs ) - this will not work well, it'll still require users to run stuff as cluster-admin if they want validation to point to the latest version. 

If we do solve the floating revisions ( i.e. use the service to point to the correct workload ) - we don't need to change anything here, leave the validation as is and when we upgrade we modify the service in istio-system.